settings.gradle.kts

```diff
@@ -17,4 +17,12 @@ dependencyResolutionManagement {
         maven(url = "https://artifactory.img.ly/artifactory/imgly")
     }
 }
+// TODO: This will be deprecated in future. Migrate to the newer `pluginManagement { includeBuild() }` mechanism instead of explicitly substituting dependency.
+//includeBuild("../Simple-Commons") {
+//    dependencySubstitution {
+//        substitute(module("com.github.SimpleMobileTools:Simple-Commons")).using(project(":commons"))
+//    }
+//}
+rootProject.name = "Simple-Gallery"
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
what are these 2 lines needed for?

Reply:
> `rootProject.name = "Simple-Gallery"`

Gradle: "Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching."

> `enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")`

Type safe project accessors allow you to replace `implementation(project(":feature:module"))` with `implementation(project.feature.module)`. It is enabled implicitly by `includedBuild()` so I just put it there.